opencdms_process/cli.py

```diff
@@ -7,8 +7,9 @@
 def main(args=None):
     """Console script for opencdms_process."""
     # See click documentation at https://click.palletsprojects.com/
-    click.echo("Replace this message by putting your code into "
-               "opencdms_process.cli.main")
+    click.echo(
+        "Replace this message by putting your code into " "opencdms_process.cli.main"
```

Comment:
Was the formatting change here accidental? It looks a bit odd being >79 chars and also the unnecessary concatenation

Reply:
Yes, it was due to black, not formatting strings. I have added a make command in the makefile with a preview flag for black so that black takes care of strings as well.